src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,8 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        StringBuilder sb = new StringBuilder();
```

Comment:
sb is not very informative, let's rename this variable

Reply:
done